ui/packages/consul-ui/app/styles/debug.scss

```diff
@@ -4,7 +4,14 @@
 // temporary component debugging setup
 @import 'consul-ui/components/main-nav-vertical/debug';
 
+.is-debug .brand-loader {
+  display: none !important;
+}
+
 .docs {
+  [role='banner'] nav:first-of-type {
+    height: calc(100vh - var(--chrome-height, 47px));
+  }
```

Comment:
@johncowen Noticed in the eng docs that the left-hand navigation had a gap at the bottom. This rule is a version of the standard one but removes the ` - 35px` portion of that calc.